styles/widgets/base/scheduler.less

```diff
@@ -813,6 +814,7 @@
         }
         .dx-scheduler-date-table-first-of-month {
             color: @SCHEDULER_WORKSPACE_ACCENT_COLOR;
+            background-color: @SCHEDULER_FIRST_OF_MONTH_CEL_BACKGROUND_COLOR;
```

Comment:
We are already within `.dx-scheduler-work-space-month` scope. Can we rename `.dx-scheduler-date-table-first-of-month` to something like `.dx-scheduler-date-table-first-cell` ?

Reply:
In my opinion, it's more correct to specify that this is the first cell of the month. And not the first cell of the month view.